library/src/main/java/com/pokegoapi/api/pokemon/EvolutionInfo.java

```diff
@@ -419,6 +419,24 @@
 	}
 
 	/**
+	 * Get evolutions
+	 *
+	 * @param pokemonId pokemon id
+	 * @return ordered evolutions
+	 */
+	public static List<PokemonId> getEvolutions(PokemonId pokemonId) {
```

Comment:
That could work as well, but I preferred to create a new collection instance so that the user has no write access to the original array instance